pkg/assets/node/configmap.go

```diff
@@ -7,11 +7,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func DaemonsetConfigMap(dsname string, nsname string, falconsensor *falconv1alpha1.FalconSensor) *corev1.ConfigMap {
-	return daemonsetConfigMap(dsname, nsname, falconsensor)
-}
-
-func daemonsetConfigMap(dsname string, nsname string, falconsensor *falconv1alpha1.FalconSensor) *corev1.ConfigMap {
```

Comment:
Let's keep this function as it allows us to minimize the amount of functions in the future if we have different types of configmaps here.

Reply:
Let's keep an extra function today, so we can minimise amount of functions in future. Please explain.